guides/hack/03-expressions-and-operators/27-function-call-examples/array-of-closures.php

```diff
@@ -17,10 +17,27 @@ function(int $p) {
     ), // times 5
   ];
```

Comment:
not your fault but while we're at it, replacing these with lambdas would significantly shorter the example without making it any less readable